hack/bin/get-webapp-tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+cd $1
+
+git checkout master
+webappversion=$(git describe --tags)
+git checkout $webappversion
+ref=$(git rev-parse --short=6 HEAD)
+# NOTE: the docker tags seem to get the app-config version wrong. But lets duplicate this bug.
+configversion=$(jq -r '.dependencies["wire-web-config-default-staging"]' ./app-config/package.json  | cut -d'#' -f2)
```

Comment:
Do we need to deal with `wire-web-config-default-master` too?

Reply:
I wrote a comment in the Backend + Web Team chat.

it seems their tags are a bit buggy